src/cpp/server/ServerPAMAuth.cpp

```diff
@@ -173,16 +173,16 @@ std::string userIdentifierToLocalUsername(const std::string& userIdentifier)
       // view of the username (as that's what the session uses to form the
       // stream path), which is why we do a username => username transform
       // here. See case 5413 for details.
-      core::system::user::User user;
-      Error error = core::system::user::userFromUsername(userIdentifier, &user);
-      if (error) 
+      core::system::User user;
+      Error error = core::system::User::createUser(userIdentifier, user);
```

Comment:
I really don't like the `createUser` method name. It makes it sound like we're actually creating a user. What's wrong with the old name?

Reply:
I was going for verbs again. I also didn't want different names for creating a user from username vs. from user ID. What about `makeUser`?

Reply:
We could use `userFromIdentifier` for username and user ID. Again, for me `make` implies some sort of actual underlying user creation.